torch/linalg/__init__.py

```diff
@@ -8,6 +8,76 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
+linalg.cholesky(input, *, out=None) -> Tensor
+
+Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
+positive-definite matrix or the Cholesky decompositions for a batch of such matrices.
+The returned matrix ``L`` is lower-triangular, and
+the decomposition has the form:
+
+.. math::
+
+    \text{input} = LL^H
+
+where :math:`L^H` is the conjugate transpose of :math:`L`, which is just a transpose for the case
+of real-valued input matrices.
+In code it translates to ``input = L @ L.t()` if :attr:`input` is real-valued and
+``input = L @ L.conj().t()`` if :attr:`input` is complex-valued.
+
+Supports real and complex inputs. Backpropagation for complex inputs is only supported on the CPU.
+
+.. note:: If :attr:`input` is not a Hermitian positive-definite matrix, or if it's a batch of matrices
+          and one or more of them is not a Hermitian positive-definite matrix, then a RuntimeError will be thrown.
+          The error message tells the index of the first problematic matrix in the batch.
+
+Args:
+    input (Tensor): the input tensor of size :math:`(*, n, n)` where `*` is zero or more
+                batch dimensions consisting of Hermitian positive-definite matrices.
```

Comment:
It's not that the batch dimensions consistent of Hermitian positive-definite matrices, it's that each matrix in the batch is expected to be Hermitian positive-definite.